src/pedestal_toolbox/params.clj

```diff
@@ -68,6 +68,21 @@
             (assoc ctx :response (response/bad-request e)))))})
     {:schema schema}))
 
+(def query-param-content-type
+  "An enter interceptor that fakes an Accept header so that later
```

Comment:
Should be "before interceptor." Yes the terminology is annoyingly inconsistent with the keywords you use to define them. But that's what they call them.